### DISCUSSION TITLE
A newbie, how to solve

### DISCUSSION THREAD

**Discussion by EDmondqin** [itzg/docker-minecraft-bedrock-server#343] [Q&A]
When I run

**docker run -d -it -e EULA=TRUE -p 19132:19132/udp itzg/minecraft-bedrock-server**

The following message will appear

**WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
74c057cefbdcd7f5f019955f7086ff99083ceb79ce625eb5f7aac57fbac451b1**

how to solve

aarch64

**Comment by itzg** [comment]
I am not able to recreate this problem on an M1 MacBook, which report as "aarch64" from `docker info`. Do a `docker pull itzg/minecraft-bedrock-server` to double check you have the latest image.
